pkg/build/util/util.go

```diff
@@ -307,3 +307,19 @@ func SafeForLoggingS2IConfig(config *s2iapi.Config) *s2iapi.Config {
 	newConfig.ScriptsURL, _ = s2iutil.SafeForLoggingURL(newConfig.ScriptsURL)
 	return &newConfig
 }
+
+// Gets the Env section of the build config regardless of the build strategy
+func GetEnvFromStrategy(build *buildapi.Build) *[]kapi.EnvVar {
```

Comment:
This is duplicating non-exported code at least in pkg/build/admission/buildpodutil.go, pkg/build/admission/defaults/admission.go and pkg/build/util/util.go.
If you end up adding an update equivalent, there's one already in pkg/build/generator/generator.go.
If this isn't dealt with in this PR, please could you open a follow-on issue?